### ISSUE TITLE
[Epic] Document Scribe-Android functions and classes

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Android#18]
### Terms

- [x] I have searched [open and closed issues](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue)
- [x] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Issue

After #17 is finished, a the codes for Scribe-Android should be more fully documented to make future work and collaboration easier. All functions and classes should be given documentation headers, and all parts of the code that are complex should be explained. Comments should also be formatted so that those that are on their own line are a complete sentence.

**Comment by andrewtavis** [comment]
Do you think it would make sense to make sub issues for this, @angrezichatterbox? If so, is there a directory structure you'd suggest where maybe we could make an issue to document the functions in a certain directory or file? We can also talk about this in the sync as well 😊

**Comment by angrezichatterbox** [comment]
> Do you think it would make sense to make sub issues for this, [@angrezichatterbox](https://github.com/angrezichatterbox)? If so, is there a directory structure you'd suggest where maybe we could make an issue to document the functions in a certain directory or file? We can also talk about this in the sync as well 😊

Yes I think it makes sense. In specific I think we have to document the services directory first. Then we could have issues for the helpers directory. These for me are the important directories we could document.

**Comment by andrewtavis** [comment]
Ok, great 😊 I'll make issues for these two then :) Thanks for your feedback!

**Comment by angrezichatterbox** [comment]
Should we add more sub-issues for this issue for the documentation part

**Comment by andrewtavis** [comment]
Yes that sounds great, @angrezichatterbox :)
